README.md

```diff
@@ -1,8 +1,53 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Nazanin Akbari . Jarvis Consulting
 
+I am a graduate of Concordia University's software engineering program, and I have recently joined Jarvis as a software and data engineer. During Master's program, I spent two years to develop multiple projects with Matlab, Python and C# in different teams of developers. I am fluent in French language. I love programming and computer science because it is a language that can apply to every facet of life. Software provides significant benefits to everything in our lives, and it's critical to have a deep knowledge of it.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Docker and Containerization, Intellij IDE, Visual Studio, Google Cloud Platform, REST API
+
+**Familiar:** Neural Network, Machine Learning, C#, Azure DevOps, Unit testing
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari](https://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/feature/creating-DDL/linux_sql)]: Developed a Linux monitoring agent that allows a user to monitor and store hardware specifications and usage information for multiple Linux systems. The agent employs Bash Scripts to collect machine hardware information, which is then stored in a PostgreSQL database provisioned using Docker. The resource usage is fetched periodically with Crontab, and we can then manipulate the aggregated data using SQL queries.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/feature/java-apps/core_java)]:
+      
+  -  Twitter App: Currently working on the project.
+  -  JDBC App: Currently working on the project.
+  - Grep App: Implemented Java Grep application which is looking for a regular expression that is searched in each file. I wrote this program in Java, using Streams, Lambda, and RegEx pattern matching, with Maven as the build tool. It is configured to be run as a command line with the JRE installed or via a docker container.
```

Comment:
- Grep App: Implemented Java Grep application which allows users to search text patterns from a Linux filesystem using core Java, Lambda expression, Maven, IntelliJ, Regex, etc. (Memory optimization ) ... docker .. docker hub.. testing?